public/jsx/views/complete-payment.jsx

```diff
@@ -18,6 +19,10 @@ module.exports = React.createClass({
     userEmail: React.PropTypes.string.isRequired,
   },
 
+  componentDidMount: function() {
+    tracking.setPage('/payment/complete');
```

Comment:
Hmm according to the docs it must start with a '/' https://developers.google.com/analytics/devguides/collection/analyticsjs/pages